src/tree_walker.js

```diff
@@ -21,7 +21,7 @@
  *     traversing.
  * @constructor
  */
-function TreeWalker(node) {
+export function TreeWalker(node) {
```

Comment:
I wasn't able to make this an `export default` because of some bug in your linter.

So now it's imported as `import {TreeWalker} from './tree-walker';` anywhere it's used.